### PR TITLE
[13.0][MIG] hr_holidays_leave_auto_approve

### PR DESCRIPTION
**Pull request by astirpe**
Includes https://github.com/OCA/hr/pull/778

Supersedes https://github.com/OCA/hr/pull/704

### CODE REVIEW

File: hr_holidays_leave_auto_approve/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2019 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Auto Approve Leaves",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "summary": "Leave type for auto-validation of Leaves",
+    "author": "Onestein, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr",
```
Comment (MiquelRForgeFlow) [inline hr_holidays_leave_auto_approve/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/hr-holidays",
```

**Comment by MiquelRForgeFlow** [comment]
?

**Comment by astirpe** [comment]
I didn't close this PR. I just forced pushed and it closed automatically. And I even don't have the Reopen button enabled....
I'll check tomorrow if I can reopen it, otherwise I will open another PR.

**Comment by alexey-pelykh** [comment]
image

**Comment by alexey-pelykh** [comment]
You pushed something wrongly, I guess

**Comment by astirpe** [comment]
Replaced by https://github.com/OCA/hr-holidays/pull/5

**Pull request closed without merging**